homeassistant/components/weblink.py

```diff
@@ -20,8 +20,9 @@
 DOMAIN = 'weblink'
 
 ENTITIES_SCHEMA = vol.Schema({
-    # pylint: disable=no-value-for-parameter
-    vol.Required(CONF_URL): vol.Url(),
+    vol.Required(CONF_URL): vol.Any(
+        vol.Match(r'\A/[a-z0-9]', msg='invalid absolut url'),
```

Comment:
"Invalid relative URL"

Reply:
It's also not matching all relative URLs. `/states/group.bla` would fail.

Reply:
thank you for your review. I'll change the text. The second one is interesting. ```/states/group.bla``` works at runtime but the test case fails.